### DISCUSSION TITLE
Proyecto

### DISCUSSION THREAD

**Discussion by jfelipeo** [rramosp/ai4eng.v1#93] [00 - General]
Buenos días profe.

Queríamos saber si este reto nos sirve para el proyecto:

https://www.kaggle.com/c/nyc-taxi-trip-duration/overview/description

Saludos.

**Comment by mksfrontierx** [comment]
hola felipe, sí, ese no está mal ... podrás si quieres integrar información geográfica adicional para tu predicción (barrios, tiempos, etc).
